internal/util/subjectsetbytype_test.go

```diff
@@ -0,0 +1,69 @@
+package util
+
+import (
+	"sort"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	core "github.com/authzed/spicedb/pkg/proto/core/v1"
+	"github.com/authzed/spicedb/pkg/tuple"
+)
+
+func RR(namespaceName string, relationName string) *core.RelationReference {
+	return &core.RelationReference{
+		Namespace: namespaceName,
+		Relation:  relationName,
+	}
+}
```

Comment:
this method is duplicated quite a lot in the codebase, especially in tests. I understand the need for making the tests readable with a short method name, but I think we should at least implement it in terms of `pkg/namespace/builder.go:RelationReference`

Reply:
its a tradeoff; some tests we do `var RR = tuple.RR` or somesuch, but either way, it should not be changed in this PR